Game/templates/WaitingRoom/WRhost.html

```diff
@@ -86,8 +86,13 @@ <h3>{{player.player_name.title}}</h3>
 
 </main>
 
-</body>
+<script>
+function timedRefresh(timeoutPeriod) {
+	setTimeout("location.reload(true);",timeoutPeriod);
+}
+window.onload = timedRefresh(1000);
```

Comment:
The time refresh of the page should be 10000 instead of 1000. It's would be better